tools/pre-swap-checker/contracts/PreSwapChecker.sol

```diff
@@ -23,6 +23,7 @@ contract PreSwapChecker {
 
   bytes4 constant internal ERC721_INTERFACE_ID = 0x80ac58cd;
   bytes4 constant internal ERC20_INTERFACE_ID = 0x36372b07;
+  bytes4 constant internal ERC1155_INTERFACE_ID= 0xd9b67a26;
```

Comment:
For what purpose is this here? It isn't used

Reply:
removed, i have another branch for erc1155 but its not part of this pr.